packages/office-ui-fabric-react/src/components/pickers/examples/Picker.CustomResult.Example.tsx

```diff
@@ -324,82 +336,56 @@ const SelectedDocumentItem: (documentProps: IPickerItemProps<IFullDocumentCardPr
-          checked={this.state.isPickerDisabled}
-          onChange={this._onDisabledButtonClick}
-        />
-        <DocumentPicker
-          removeButtonAriaLabel="Remove"
-          onRenderSuggestionsItem={SuggestedBigItem as any}
-          onResolveSuggestions={this._onFilterChanged}
-          onRenderItem={SelectedDocumentItem}
-          getTextFromItem={this._getTextFromItem}
-          pickerSuggestionsProps={{
-            suggestionsHeaderText: 'Suggested Documents',
-            noResultsFoundText: 'No Documents Found',
-          }}
-          disabled={this.state.isPickerDisabled}
-          inputProps={{
-            onFocus: () => console.log('onFocus called'),
-            onBlur: () => console.log('onBlur called'),
-            'aria-label': 'Document Picker',
-          }}
-        />
-      </div>
-    );
-  }
-
-  private _getTextFromItem(props: any): any {
-    return props.documentTitleProps.title;
-  }
+  const getTextFromItem = useConstCallback((props: IFullDocumentCardProps): string => {
+    return props.documentTitleProps!.title;
+  });
 
-  private _onDisabledButtonClick = (): void => {
-    this.setState({
-      isPickerDisabled: !this.state.isPickerDisabled,
-    });
-  };
-
-  private _onFilterChanged(filterText: string, items?: IFullDocumentCardProps[]): IFullDocumentCardProps[] {
-    if (!items) {
-      return [];
-    }
-
-    return filterText
-      ? data
-          .filter(
-            item =>
-              item.documentTitleProps &&
-              item.documentTitleProps.title.toLowerCase().indexOf(filterText.toLowerCase()) === 0,
-          )
-          .filter(item => !this._listContainsDocument(item, items))
-      : [];
-  }
-
-  private _listContainsDocument(document: IFullDocumentCardProps, items: IFullDocumentCardProps[]): boolean {
+  const listContainsDocument = (document: IFullDocumentCardProps, items: IFullDocumentCardProps[]): boolean => {
     if (!items || !items.length || items.length === 0) {
       return false;
     }
     const documentTitle = document.documentTitleProps && document.documentTitleProps.title;
-    return (
-      items.filter(item => (item.documentTitleProps && item.documentTitleProps.title) === documentTitle).length > 0
-    );
-  }
-}
+    return items.some(item => (item.documentTitleProps && item.documentTitleProps.title) === documentTitle);
+  };
+
+  const onFilterChanged = useConstCallback(
+    (filterText: string, items?: IFullDocumentCardProps[]): IFullDocumentCardProps[] => {
+      if (!items) {
+        return [];
+      }
+      return filterText
+        ? data
+            .filter(
+              item =>
+                item.documentTitleProps &&
+                item.documentTitleProps.title.toLowerCase().indexOf(filterText.toLowerCase()) === 0,
+            )
+            .filter(item => !listContainsDocument(item, items))
+        : [];
+    },
+  );
+
+  return (
+    <div className={rootClass}>
+      <Checkbox
```

Comment:
This really ought to be a toggle, not a checkbox, but that's an issue in a lot of the examples and doesn't necessarily have to be fixed now.